src/h2load.cc

```diff
@@ -75,7 +75,7 @@ Config::Config()
     : data_length(-1), addrs(nullptr), nreqs(1), nclients(1), nthreads(1),
       max_concurrent_streams(-1), window_bits(30), connection_window_bits(30),
       no_tls_proto(PROTO_HTTP2), data_fd(-1), port(0), default_port(0),
-      verbose(false) {}
+      verbose(false), nconns(0), rate(0), current_worker(0) {}
```

Comment:
The fields in initializer list must be ordered as they appear in struct.  I got following warnings:
```
h2load.cc:78:7: warning: field 'verbose' will be initialized after field 'nconns'
      [-Wreorder]
      verbose(false), nconns(0), rate(0), current_worker(0) {}
      ^
h2load.cc:78:23: warning: field 'nconns' will be initialized after field 'rate'
      [-Wreorder]
      verbose(false), nconns(0), rate(0), current_worker(0) {}
                      ^
```